checkout/src/main/java/com/checkout/logging/EventLogger.kt

```diff
@@ -18,6 +18,9 @@ internal class EventLogger(private val logger: CheckoutEventLogger) : Logger<Log
     @VisibleForTesting
     var needToSetup = true
 
+    @VisibleForTesting
```

Comment:
let's remove this as the field is meant to be accessible, not just for testing.

also it's worthy adding unit tests to `EventLoggerTest` so that when `resetSession` is called, the `correlationId` is same as the value called in `logger.addMetadata(METADATA_CORRELATION_ID, correlationId)`